tools/testing/gatekeeper/required-tests.yaml

```diff
@@ -161,36 +161,36 @@ mapping:
       - Static checks / check-kernel-config-version
       - Static checks / static-checks (make static-checks)
       # static-checks-self-hosted.yaml
-      - Static checks self-hosted / build-checks (s390x) / check (make check, agent-ctl, src/tools/agent-ctl, rust, protobuf-compiler, clang, s390x)
```

Comment:
I think the `build-checks (s390x)` is the name of the runner, so all the `ubuntu-24.04-s390x` changes need fixing here too e.g. https://github.com/kata-containers/kata-containers/actions/runs/19573103903/job/56051289676?pr=12096#logs